modules/post/windows/manage/pptp_tunnel.rb

```diff
@@ -22,7 +22,8 @@ def initialize(info = {})
         'License' => MSF_LICENSE,
         'Author' => 'Borja Merino <bmerinofe[at]gmail.com>',
         'References' => [
-          [ 'URL', 'https://www.youtube.com/watch?v=vdppEZjMPCM&hd=1' ]
+          [ 'URL', 'https://www.youtube.com/watch?v=vdppEZjMPCM&hd=1' ],
+          [ 'ATT&CK', Mitre::Attack::Technique::T1021_REMOTE_SERVICES ]
```

Comment:
This seems to be sort of a VNC relay attack, it might be better to use the VNC sub-category?